client/src/components/Post/Post.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+.dots {
```

Comment:
pls add wrapper to component and add nesting

Reply:
I cannot do that because style lint will be arguing that maximum nesting depth exceeded which is 2